### PR TITLE
add docsify-image-caption

### PR DESCRIPTION
**Pull request by h-hg**
docs: ::memo:: Add docsify-image-caption to Plugins, a plugin to show image caption in docsify.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -207,6 +207,7 @@ A curated list of awesome things related to <a href="//docsify.js.org">docsify</
 - [docsify-updated](https://github.com/pfeak/docsify-updated) - Add update time for each of your docsify markdown.
 - [docsify-ko-fi](https://github.com/fcannizzaro/docsify-ko-fi) - A Docsify plugin to render customized Ko-fi buttons from markdown.
 - [docsify-waline](https://github.com/cxcn/docsify-waline) - A Docsify comment plugin for docsify that supports visitor statistics.
+- [docsify-image-catpion](https://h-hg.github.io/docsify-image-catpion) - A plugin to show image caption. [@h-hg](https://github.com/h-hg/docsify-image-catpion)
```
Comment (sy-records) [inline README.md:210]: The link is not accessible.

**Review by sy-records — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (h-hg) [reply README.md]: Thanks for your comment. I have fixed the error

**Review by Koooooo-7 — APPROVED** [review]

**Review by sy-records — APPROVED** [review]
If you submit it to npm that is even better.

**Comment by h-hg** [comment]
> If you submit it to npm that is even better.

Thanks for your suggestion.

**Pull request merged**